google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CopyBackupRequest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.admin.v2.models;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.bigtable.admin.v2.internal.NameUtil;
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+import com.google.protobuf.util.Timestamps;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import org.threeten.bp.Instant;
+
+/** Fluent wrapper for {@link com.google.bigtable.admin.v2.CopyBackupRequest} */
```

Comment:
Please add more documentation here explaining what this does. Currently protobufs are considered an internal implementation detail, so we shouldnt direct users to them. The javadoc here should be able to stand on its own

Reply:
Added example in the method doc.